meetings/2024/README.md

```diff
@@ -14,6 +14,7 @@ All schedule items must have a public issue or checked-in proposal that can be l
 - [BinaryCompatOnlyAttribute](https://github.com/dotnet/csharplang/pull/7707) (Fred)
 - Discriminated Union WG (Matt)
 - Extensions (Julien and Mads (after Feb 5))
+- [Partial type inference](https://github.com/dotnet/csharplang/pull/7582) (Rikki / [Tomas](https://github.com/TomatorCZ))
```

Comment:
This is a potential C# 13 feature so would like to see it put ahead of say WG which is C# 14+. How do we reflect that in the data here?

Reply:
Perhaps listing it after BinaryCompatOnly would make sense?